### ISSUE TITLE
Cannot get actions in nodeos 1.8.1

### ISSUE THREAD

**Issue by psychoplasma** [EOSIO/eos#7788]
After full sync where `history_api_plugin` is enabled (also the dependency plug-ins) with some filtered-on accounts, if I transfer tokens from-to those filtered-on accounts and try to `get actions` for them, `nodeos` returns the actions until the transaction becomes irreversible, but once the transaction becomes irreversible, `nodeos` does not return any actions for those accounts any more. Here are some values from the configuration file that I think that they might be related;

```
validation-mode = full
read-mode = head
.
.
filter-on = someaccount1:transfer:
filter-on = someaccount2:transfer:
.
.
plugin = eosio: http_plugin
plugin = eosio: chain_plugin
plugin = eosio: chain_api_plugin
plugin = eosio: net_api_plugin
plugin = eosio: history_plugin
plugin = eosio: history_api_plugin
```

Wondering if it's a bug or I'm setting something wrong in the configuration file.

**Comment by taokayan** [comment]
```history_plugin``` & ```history_api_plugin``` are deprecated. I guess you should try ```state_history_plugin``` instead.

**Comment by psychoplasma** [comment]
So that means `get actions` will also be deprecated? Because in documentations, it's stated that if `state_history_plugin` is used, `postgresql` is required to fetch historical data for the filtered accounts, not `get actions`. And is there any way to access `state_history_plugin` directly instead of using `postgresql`? I mean the plugin listens to `8080` port but I couldn't find any API documentation for `state_history_plugin`.

**Comment by firesWu** [comment]
In v1.8 , logs default level is `info`,  so it can't print dlog.  so that `get actions` can't print data when the actions becomes irreversible.  you can add option `-j`, `cleos get actions eosio.token -j` is normal，
```
// cleos/main.cpp
             if( trace["block_num"].as_uint64() <= lib ) {
               // you can change `dlog` to `elog`, and then recompiler.
                 dlog( "\r${m}", ("m",out.str()) );
              } else {
                 wlog( "\r${m}", ("m",out.str()) );
              }
```

**Comment by heifner** [comment]
This is fixed in `v1.8.3` via https://github.com/EOSIO/eos/pull/7869
